### PR TITLE
add tail inotify debug fix and stage 2.31.10

### LINKED ISSUE #639: Tail Debug Logging Crash issue in 2.31.7 - 2.31.9
This only affects you if ALL of following are true:

1. You are using 2.31.7 to 2.31.9
2. You are using the tail input: https://docs.fluentbit.io/manual/pipeline/inputs/tail
3. You have enabled debug logging: https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#enable-debug-logging

### Versions Impacted

2.31.7, 2.31.8 and 2.31.9

### Mitigations

1. Use a non-impacted version, we recommend 2.31.6 or 2.31.7
2. Do not enable debug logging: https://github.com/aws/aws-for-fluent-bit/blob/mainline/troubleshooting/debugging.md#enable-debug-logging
3. Do not use the tail input: https://docs.fluentbit.io/manual/pipeline/inputs/tail

### Root Cause

Bad code introduced here: https://github.com/PettitWesley/fluent-bit/commit/ab11d1d7438f5254d04eba8dfc2f10b78cc2244d

Introduced in AWS for Fluent Bit here: https://github.com/aws/aws-for-fluent-bit/commit/612f2460140d66aec5219d90aee341113eed09b9

```
flb_sds_printf(&buf, "inode=%"PRIu64", %s, events: ", file->name, file->inode);
```

The arguments are out of order, file->inode should be first, file->name should be second.

### PR DESCRIPTION
**Pull request by PettitWesley**
*Issue #, if available:*

fixes #639 #637 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by PettitWesley** [comment]
Already fixed in upstream Fluent Bit: https://github.com/fluent/fluent-bit/commit/3e360f42feb2e5d9de45258afb20630177f61127

**Review by matthewfala — APPROVED** [review]
Looks good! thank you.

File: init/fluent_bit_init_process_test.go [hunk 1]
```diff
@@ -59,23 +60,45 @@ func TestGetECSTaskMetadata(t *testing.T) {
 		ECS_TASK_ARN:          "arn:aws:ecs:us-west-2:123456789123:task/ecs-cluster/4ca5a280e68947cd84a8357f0d008fb5",
 		ECS_TASK_ID:           "4ca5a280e68947cd84a8357f0d008fb5",
 		ECS_FAMILY:            "code_test_A",
-		ECS_LAUNCH_TYPE:       "", // empty lunch type
```
Comment (matthewfala) [inline init/fluent_bit_init_process_test.go:62]: 😄 🥪

**Review by matthewfala — APPROVED** [review]
New changes look good.

**Pull request merged**